carbon-design-system/carbon
# fix(combo-box): support v10 input controls

Closes #2240

This PR adds styles for the v10 combo box selection button and refactors multiselect selection button styles

Related:

https://github.com/IBM/carbon-components-react/issues/2109

https://github.com/IBM/carbon-components-react/issues/2113

#### Changelog

**New**

- add styles for v10 combo box input controls

**Changed**

- nest warning icons within list box field

#### Testing / Reviewing

- Ensure listbox is not visually broken
- Ensure multiselect is not visually broken
- Ensure combobox is not visually broken
